build_daemon/test/daemon_test.dart

```diff
@@ -128,27 +138,37 @@ void main() {
   });
 }
 
-Future<String> getOutput(Process daemon) async {
-  return await daemon.stdout
-      .transform(utf8.decoder)
-      .transform(const LineSplitter())
-      .firstWhere((line) => line == 'RUNNING' || line == 'ALREADY RUNNING');
-}
+Future<String> getOutput(Process daemon) async =>
+    await daemon.stdout
```

Comment:
This is pretty difficult to follow with the indentation that dartfmt chose and the size of the expressions. I would create some local variables to make it more clear.